tests/difference/core/quint_model/README.md

```diff
@@ -0,0 +1,72 @@
+This folder contains a Quint model for the core logic of Cross-Chain Validation (CCV).
+
+### File structure
+The files are as follows:
+- ccv.qnt: Contains the type definitions, data structures, functional logic for CCV.
+The core of the protocol.
+- ccv_statemachine.qnt: Contains the state machine layer for CCV. Very roughly speaking, this could be seen as "e2e tests".
+Also contains invariants.
+- ccv_test.qnt: Contains unit tests for the functional layer of CCV.
+- libararies/*: Libraries that don't belong to CCV, but are used by it.
+
+### Model details
+
+To see the data structures used in the model, see the ProtocolState type in ccv.qnt.
+
+The "public" endpoints of the model are those functions that take as an input the protocol state, and return a Result.
+Other functions are for utility.
+
+The parameters of the protocol are defined as consts in ccv.qnt.
+
+### Tests
+
+To run unit tests, run 
+```
+quint test ccv_test.qnt --main CCVTest
+```
+and 
+```
+quint test ccv_statemachine.qnt --main CCVDefaultStateMachine
```

Comment:
This one fails on my machine:
```
quint test ccv_statemachine.qnt --main CCVDefaultStateMachine

quint test <input>

Run tests against a Quint specification

Options:
  --help         Show help                                             [boolean]
  --version      Show version number                                   [boolean]
  --out          output file (suppresses all console output)            [string]
  --main         name of the main module (by default, computed from filename)
                                                                        [string]
  --output       write a trace for every test, e.g., out{#}{}.itf.json
                 {} is the name of a test, {#} is the test sequence number
                                                                        [string]
  --max-samples  the maximum number of successful runs to try for every
                 randomized test                       [number] [default: 10000]
  --seed         random seed to use for non-deterministic choice        [string]
  --verbosity    control how much output is produced (0 to 5)
                                                           [number] [default: 2]
  --match        a string or regex that selects names to use as tests   [string]

Error: Couldn't find any result for id 53 while Inferring effect for ConsumerChains
    at EffectInferrer.fetchResult (/opt/homebrew/lib/node_modules/@informalsystems/quint/dist/src/effects/inferrer.js:262:19)
    at EffectInferrer.effectForName (/opt/homebrew/lib/node_modules/@informalsystems/quint/dist/src/effects/inferrer.js:278:25)
    at EffectInferrer.exitName (/opt/homebrew/lib/node_modules/@informalsystems/quint/dist/src/effects/inferrer.js:99:41)
    at walkExpression (/opt/homebrew/lib/node_modules/@informalsystems/quint/dist/src/IRVisitor.js:272:25)
    at /opt/homebrew/lib/node_modules/@informalsystems/quint/dist/src/IRVisitor.js:289:38
    at Array.forEach (<anonymous>)
    at walkExpression (/opt/homebrew/lib/node_modules/@informalsystems/quint/dist/src/IRVisitor.js:289:23)
    at /opt/homebrew/lib/node_modules/@informalsystems/quint/dist/src/IRVisitor.js:289:38
    at Array.forEach (<anonymous>)
    at walkExpression (/opt/homebrew/lib/node_modules/@informalsystems/quint/dist/src/IRVisitor.js:289:23)
```

Reply:
Could you check that your quint is up-to-date? 
```
$ quint --version 
0.14.4
```

Reply:
Thanks Philip. This was a personal setup issue on my side. Reinstalled quint and it works.